### DISCUSSION TITLE
Cleanlab for Object Detection

### DISCUSSION THREAD

**Discussion by Giriteja** [cleanlab/cleanlab#599] [General]
Any plans to release how to use Cleanlab for Object detection?Or can you tell me how to use existing clean lab functionalities for object detection usecase

**Comment by jwmueller** [comment]
Yes! This is our most requested feature and our team is actively working on it!  Our roadmap is publicly available here: https://github.com/orgs/cleanlab/projects/2/views/1

I'll ping you here once we have merged code in with dedicated functionality for finding label errors in object detection datasets.

For now, here is a crude solution that can be used for segmentation datasets which some users have also successfully adapted for object detection applications:  https://github.com/cleanlab/cleanlab/issues/586

**Comment by michen00** [comment]
Does #676 address this? If so, are there any tutorials or example code snippets I can reference yet?

**Reply by michen00** [reply]
Nvm, I found the notebook:
https://github.com/cleanlab/cleanlab/blob/master/docs/source/tutorials/object_detection.ipynb

**Reply by jwmueller** [reply]
yep! you can see also this version of the same notebook rendered on our website: https://docs.cleanlab.ai/master/tutorials/object_detection.html

**Reply by michen00** [reply]
It was indeed helpful to have the visualizations rendered, thanks!
